app/app/(app-active-cadets)/active-cadets/page.tsx

```diff
@@ -0,0 +1,4 @@
+const ActiveCadets = () => {
```

Comment:
For page components name 

It's recommended to have word `Page`
For the Page component 
e.g. `ActiveCadetPage`

And Page component should be a normal
function

`export default function ... Page () ... `

Reply:
Let's create NORM.md

Reply:
https://github.com/42-Bangkok/portal/pull/54

Reply:
changed to  ActiveCadetPage